packages/strapi-admin/ee/admin/hooks/useAuthProviders/index.js

```diff
@@ -4,15 +4,25 @@ import { request } from 'strapi-helper-plugin';
 import { getRequestUrl } from '../../../../admin/src/utils';
 import reducer, { initialState } from './reducer';
 
-const useAuthProviders = () => {
+const useAuthProviders = ({ ssoEnabled } = {}) => {
```

Comment:
Nitpick: not sure we need the initial value anymore

Reply:
updated